build.gradle

```diff
@@ -7,8 +7,9 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.1.0'
         classpath 'com.dicedmelon.gradle:jacoco-android:0.1.2'
+        classpath 'org.codehaus.groovy:groovy-all:2.4.12'
```

Comment:
Workaround for https://github.com/arturdm/jacoco-android-gradle-plugin/issues/42